src/api-usability.bib

```diff
@@ -0,0 +1,61 @@
+@article{myers2016improving,
+  title={Improving API usability},
+  author={Myers, Brad A and Stylos, Jeffrey},
+  journal={Communications of the ACM},
+  year={2016},
+}
+
+
+@article{scheller2015automated,
+  title={Automated measurement of API usability: The API concepts framework},
+  author={Scheller, Thomas and K{\"u}hn, Eva},
+  journal={Information and Software Technology},
+  year={2015},
+}
+
+
+@inproceedings{stylos2007usability,
+  title={Usability implications of requiring parameters in objects' constructors},
+  author={Stylos, Jeffrey and Clarke, Steven},
+  booktitle={Proceedings of the 29th international conference on Software Engineering},
```

Comment:
Capitalize "International" and "Conference" (don't need braces outside of the title)